deploy/crds/crd-certificates.yaml

```diff
@@ -196,6 +196,8 @@ spec:
                     size:
                       description: Size is the key bit size of the corresponding private key for this certificate. If `algorithm` is set to `RSA`, valid values are `2048`, `4096` or `8192`, and will default to `2048` if not specified. If `algorithm` is set to `ECDSA`, valid values are `256`, `384` or `521`, and will default to `256` if not specified. If `algorithm` is set to `Ed25519`, Size is ignored. No other values are allowed.
                       type: integer
+                rawSubject:
+                  type: string
```

Comment:
It seems a bit weird that there's no description here. Can we add one?

Reply:
I renamed the field to `literalSubject` and added the description but it is gone after running the `hack/update-all.sh` script. Does the `description` come from the comment?